src/v/transform/tests/BUILD

```diff
@@ -0,0 +1,26 @@
+load("//bazel:test.bzl", "redpanda_cc_gtest")
+
+redpanda_cc_gtest(
+    name = "transform_commit_batcher_test",
```

Comment:
You don't need to prefix with transform anymore because there is no global namespace of binaries.

Reply:
🤯

Reply:
>You don't need to prefix with transform anymore because there is no global namespace of binaries.

Ahh makes sense. I need to update many names I think then. I'm losing steam on addressing some of these aesthetics.

Reply:
Its fine more of an fyi